PendingReleaseNotes.md

```diff
@@ -44,4 +45,9 @@ ConfigMap can be used in mix with the already existing Env Vars defined in opera
 
 ## Deprecations
 
+### Ceph
+
+- The CR spec value `hostNetwork` is now deprecated in favor of `provider: host`.
```

Comment:
We can't remove the `hostNetwork` since that would be a breaking change for a commonly used feature. We need to maintain support for this property. We could keep support for this property, but recommend new clusters start using the new setting. Internally, we would need a helper to check for host networking and still read `hostNetwork: true`. Maybe the helper `IsHost()` will already help with that pattern.

Reply:
It's a wording issue, `hostNetwork` is not gone, you can still use it but `provider: host` is preferred. No sweat :)